docs/source/modin/supported/resampling_supported.rst

```diff
@@ -76,15 +76,13 @@ Computations / descriptive stats
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``min``                     | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``nunique``                 | N                               |                                                    |
+| ``nunique``                 | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``ohlc``                    | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``pad``                     | N                               |                                                    |
```

Comment:
`Resampler.pad` does not exist in pandas docs: https://pandas.pydata.org/docs/reference/resampling.html. Having it here was a bug in the documentation.

Reply:
Makes sense. Thanks for removing!